tensorflow/contrib/tensor_forest/kernels/v4/fertile-stats-resource.h

```diff
@@ -64,10 +64,13 @@ class FertileStatsResource : public ResourceBase {
   // node's fertile slot's statistics if or initializes a split candidate,
   // where applicable.  Returns if the node is finished or if it's ready to
   // allocate to a fertile slot.
+  // With pre-generated feature_per_split, so it's
```

Comment:
May be change to "When pre-generated feature_per_split is provided, this method is thread safe?"